contributed_definitions/nyaml/NXcoordinate_system_set.yaml

```diff
@@ -88,138 +88,259 @@ doc: |
   always guide the user for each group and field which of the coordinate system
   one refers to.
 type: group
+# express conventions explicitly and understandable
+# appdefs each instance of NXentry should have its own set of conventions
+# method-specific additional conventions on top of the ones formulated here
+# should go to method-specific base classes like NXcoordinate_system_em_ebsd
+# use depends_on field - not attribute - to point to conventions which in specific cases
 NXcoordinate_system_set(NXobject):
+  rotation_handedness(NX_CHAR):
+    doc: |
+      Convention how a positive rotation angle is defined when viewing
+      from the end of the rotation unit vector towards its origin,
+      i.e. in accordance with convention 2 of
+      DOI: 10.1088/0965-0393/23/8/083501.
+      Counter_clockwise is equivalent to a right-handed choice.
+      Clockwise is equivalent to a left-handed choice.
+    enumeration: [undefined, counter_clockwise, clockwise]
+  rotation_convention(NX_CHAR):
+    doc: |
+      How are rotations interpreted into an orientation
+      according to convention 3 of
+      DOI: 10.1088/0965-0393/23/8/083501.
+    enumeration: [undefined, passive, active]
+  euler_angle_convention(NX_CHAR):
+    doc: |
+      How are Euler angles interpreted given that there are several
+      choices (e.g. ZXZ, XYZ, etc.) according to convention 4 of
+      DOI: 10.1088/0965-0393/23/8/083501.
+      The most frequently used convention is ZXZ which is based on
+      the work of H.-J. Bunge but other conventions are possible.
+    enumeration: [undefined, zxz]
```

Comment:
if there are several choices (you even mention some above), they should be reflected here.

Reply:
Good point, implemented